theano/compile/function.py

```diff
@@ -115,10 +118,10 @@ def function(inputs, outputs=None, mode=None, updates=None, givens=None,
         Ops in the graph, an Exception will be raised.
     allow_input_downcast: bool or None
         True means that the values passed as inputs when calling the function
-        can be silently downcasted to fit the dtype of the corresponding
+        can be silently down-casted to fit the dtype of the corresponding
```

Comment:
My english isn't perferct. I have found downcast in dictionary by googling. I don't have a preference. @lamblin answered for the other comments. I like is suggestion.